src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -482,7 +484,10 @@ class ClientChannel::SubchannelWrapper : public SubchannelInterface {
       GPR_ASSERT(it != chand_->subchannel_refcount_map_.end());
       --it->second;
       if (it->second == 0) {
-        chand_->channelz_node_->RemoveChildSubchannel(subchannel_node->uuid());
+        if (chand_->channelz_node_ != nullptr) {
```

Comment:
Similar thing here: this conditional can actually wrap all of the code from lines 481 to 493.

Reply:
Done